src/app/models/field/emailField.ts

```diff
@@ -29,9 +29,15 @@ const createEmailFieldSchema = (): Schema<IEmailFieldSchema> => {
       includeFormSummary: {
         type: Boolean,
         default: false,
-        set: function (this: IEmailFieldSchema, v: boolean) {
-          // PDF response not allowed for encrypt forms
-          return this.parent().responseMode === ResponseMode.Encrypt ? false : v
+        validate: {
```

Comment:
there's a pre-validate hook below in the code which seems to do the same thing. that can be deleted, right?